### ISSUE TITLE
[A11y]

### ISSUE THREAD

**Issue by wallyOSU** [OregonDigital/OD2#3272]
### Operating System

Windows

### Browser

Chrome

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Tested with NVDA screen reader. Reader does not announce content of warning box - you are only told there is a warning box, with a close button. Video (sound on) demonstrates issue starting at 02:35: https://media.oregonstate.edu/media/t/1_fsbm6swq?st=155

![Image](https://github.com/user-attachments/assets/ffb8d936-04d5-418b-b599-fa7ff7344cac)

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

1. Start the screen reader
2. Attempt to set yourself as proxy
3. Ensure the screen reader announces the error text as well as the error button

**Comment by wallyOSU** [comment]
To clarify the issue - the screen reader will announce the text if you know to hover your mouse over it, but nothing announces that there is text to find, and it won't read it at all if you are navigating with a keyboard.
